coalib/coala_json.py

```diff
@@ -28,12 +31,38 @@ def main():
     args = arg_parser.parse_args()
 
     log_printer = None if args.text_logs else ListLogPrinter()
-    results, exitcode, _ = run_coala(log_printer=log_printer, autoapply=False)
+    JSONEncoder = create_json_encoder(use_relpath=args.relpath)
+
+    if args.show_bears:
+        try:
+            local_bears, global_bears = get_filtered_bears(
+                args.filter_by_language, log_printer)
 
-    retval = {"results": results}
+            bears = inverse_dicts(local_bears, global_bears)
+            results = []
+            for bear, sections in sorted(bears.items(),
+                                         key=lambda bear_tuple:
+                                         bear_tuple[0].name):
+                metadata = bear.get_metadata()
+                results.append({"Name": bear.name,
+                                "Description": metadata.desc,
+                                "Supported languages": bear.LANGUAGES,
```

Comment:
`set` is not a problem as [this](https://github.com/coala-analyzer/coala/blob/master/coalib/output/JSONEncoder.py#L23) takes care of it.
`<class 'property'>` is the problem. More specifically https://github.com/coala-analyzer/coala/blob/master/coalib/bears/Bear.py#L386

FWIW I tired commenting out the whole function but then I got another TypeError stating `<class '_sre.SRE_Pattern'>`(which is the compiled regex object) can't be serialized